### ISSUE TITLE
Typo in the License document

### ISSUE THREAD

**Issue by idimopoulos** [SEMICeu/ADMS-AP#6]
In American English, the word `licence` is not used. They only use License. It seems that DCAT was written based to the American English standards.

On the other side, it seems that ADMS-AP used the UK style grammar which is using `licence` as a noun and `license` as a verb. Under those standards, the published ADMS-AP is having a typo in every reference to the DCAT property `LicenseDocument` as it is referred as `LicenceDocument`.

References: 
[Distribution license](https://www.w3.org/TR/vocab-dcat/#Property:distribution_license)
[dct:LicenseDocument](http://purl.org/dc/terms/LicenseDocument)

We should change every instance of dct:LicenceDocument to dct:LicenseDocument in the ADMS-AP ontology.

**Comment by idimopoulos** [comment]
By the way, this fix should be applied also to references towards the license document, like the distribution license term which is currently defined as 
```
<rdf:Property rdf:about="http://purl.org/dc/terms/licence">
    <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#ObjectProperty"/>
    <dc:identifier xml:lang="en">dct:licence</dc:identifier>
    <rdfs:comment xml:lang="en">Conditions or restrictions for (re)use of the Distribution; if there are multiple licences, the most 'open' one should be given. Whenever relevant, the URIs under the concept scheme identified by http://purl.org/adms/OSIlicence/ can be used.</rdfs:comment>
    <rdfs:isDefinedBy rdf:resource="http://purl.org/dc/terms/"/>
    <rdfs:label xml:lang="en">Asset Distribution - licence</rdfs:label>
    <rdfs:range rdf:resource="http://purl.org/dc/terms/LicenceDocument"/>
    <rdfs:domain rdf:resource="http://www.w3.org/ns/dcat#Distribution"/>
    <ns1:usageNote xml:lang="en">Mandatory Property - 1..1</ns1:usageNote>
  </rdf:Property>
```
The correct one is `http://purl.org/dc/terms/license`.

**Comment by idimopoulos** [comment]
Closing this as it is fixed in version 2.02 as pointed in https://github.com/SEMICeu/ADMS-AP/issues/3
